src/core/server/rendering/views/template.tsx

```diff
@@ -250,7 +273,10 @@ export const Template: FunctionComponent<Props> = ({
               })}
             >
               {i18n('core.ui.welcomeMessage', {
-                defaultMessage: `Loading ${injectedMetadata.branding.applicationTitle}`,
+                // Wazuh: change the default message to
+                // avoid "Loading Wazuh" duplication.
+                defaultMessage: 'Loading ...',
+                // defaultMessage: `Loading ${injectedMetadata.branding.applicationTitle}`
```

Comment:
**thought**: I guess we could remove this line if it is unneeded.

Reply:
Solved